ppx/ppx.ml

```diff
@@ -943,9 +943,7 @@ let jsxMapper () =
         match str_label with
         | Str_label.Optional str ->
             [%expr
-              maybe
-                [%e Exp.ident { loc; txt = Ldot (Lident "Prop", str) }]
-                [%e value]]
+              Prop.(maybe [%e Exp.ident { loc; txt = Lident str }]) [%e value]]
```

Comment:
The previous implementation was more conservative right? Could not there be shadowing issues down the line caused by opening of `Prop` fully?

I would rather keep the ppx super accurate with namespacing, avoiding usages of `open` as much as possible:

```suggestion
             Prop.maybe
                [%e Exp.ident { loc; txt = Ldot (Lident "Prop", str) }]
                [%e value]]
```

Reply:
These seem exactly equivalent to me. `Prop.maybe Prop.foo value`, `Prop.(maybe foo) value` and `(let open Prop in maybe foo) value` all do the exact same thing. But if you think this change makes it clearer I'd be happy to accept it.